--- ./qt-everywhere-opensource-src-4.8.6/src/gui/kernel/qcocoaapplicationdelegate_mac.mm	2014-06-12 10:20:38.000000000 +0300
+++ ./qt-everywhere-opensource-src-4.8.6/src/gui/kernel/qcocoaapplicationdelegate_mac.mm	2014-06-12 21:59:18.000000000 +0300
@@ -122,7 +122,7 @@
	 [dockMenu release];
	 [qtMenuLoader release];
	 if (reflectionDelegate) {
-        [NSApp setDelegate:reflectionDelegate];
+        [[NSApplication sharedApplication] setDelegate:reflectionDelegate];
		 [reflectionDelegate release];
	 }
	 [super dealloc];
